nix/deku.nix

```diff
@@ -68,28 +68,28 @@ in ocamlPackages.buildDunePackage rec {
     ]
     # checkInputs are here because when cross compiling dune needs test dependencies
     # but they are not available for the build phase. The issue can be seen by adding strictDeps = true;.
-    ++ checkInputs
-    # some benchmarking libraries are broken om m1, so we make them optional
```

Comment:
This should have been fixed in upstream so this should be OK to remove

Reply:
Can't test this as I don't have aarch64-darwin.